src/en/tutorial-client-basic.md

```diff
@@ -0,0 +1,367 @@
+
+As you gain more experience with the client, you will encounter command
+prefixes whose meanings may not be immediately apparent.
+
+A good example of this are the `destroy-` and `remove-` prefixes, such as in
+the commands `destroy-model` and `remove-user`. These two prefixes differ in
+terms of severity. Generally speaking, `destroy-` indicates a destructive
+action that is difficult to reverse whereas `remove-` does not.
+
+There is an even more drastic-sounding "remove" prefix: `kill-`, but it is
+featured in a sole command: `kill-controller`. The latter command differs
+from `destroy-controller` in that it has the ability to terminate the
+controller machine directly via the cloud provider, without cleaning up any
+machines that may be running in workload models.
+
+The `show-` prefix is used to drill down into an object to reveal details. It
+is akin to the verb "describe".
+
+There are command aliases, that typically employ a prefix, that point to their
+corresponding commands. This practice is for simplifying the life of the
+operator. The `list-` alias/prefix is a good example, where `list-clouds`
+points to `clouds`.
+
+The list of aliases can be obtained in this way:
+
+```bash
+juju help commands | grep Alias
+```
+
+## Fundamental commands
+
+Juju makes available to the operator a wide variety of commands. In this
+section, we'll cover the basic ones. 
+
+### Listing initial information
+
+List the controller:
+
+```bash
+juju controllers
+```
+
+This will return a list of all the controllers known to your Juju client.
+Below, we have a controller named 'lxd':
+
+```no-highlight
+Controller  Model    User   Access     Cloud/Region         Models  Machines    HA  Version
+lxd*        default  admin  superuser  localhost/localhost       2         1  none  2.5.0
+```
+
+A newly-created controller has two models: The 'controller' model, which should
+be used only for internal Juju management, and a 'default' model, which is
+ready for actual use.
+
+Confirm the above by listing all the models in the currently active controller:
+
+```bash
+juju models
+```
+
+Our example's output:
+
+```no-highlight
+Controller: lxd
+
+Model       Cloud/Region         Status     Machines  Access  Last connection
+controller  localhost/localhost  available         1  admin   just now
+default*    localhost/localhost  available         2  admin   2019-02-02
+```
+
+To see the currently active controller, model, and user:
+
+```bash
+juju whoami
+```
+
+Our example:
+
+```no-highlight
+Controller:  lxd
+Model:       default
+User:        admin
+```
+
+### Deploying an application
+
+Applications are contained within models and are installed via *charms*. By
+default, charms are downloaded from the online [Charm Store][charm-store]
+during deployment.
+
+To deploy a charm, such as 'redis', in the currently active model:
+
+```bash
+juju deploy redis
+```
+
+This causes a similarly-named application ("redis") to be installed on a
```

Comment:
I don't think we want to have this appear happen stance with "similarly-named" but note more explicitly "by default the application will be named after the charm"

Reply:
OK